src/plstring.c

```diff
@@ -0,0 +1,540 @@
+#include <nitro.h>
+#include <string.h>
+
+#include "struct_decls/struct_plstring_decl.h"
+
+#include "heap.h"
+#include "plstring.h"
+
+// Important character constants
+#define CR                      0xe000
+#define TERMINATOR              0xffff
```

Comment:
Prefix the characters with `CHAR` e.g. like in [pokeemerald](https://github.com/pret/pokeemerald/blob/master/gflib/characters.h). These constants probably need to be in a global include alongside the other character constants.